libs/sdk-bindings/src/breez_sdk.udl

```diff
@@ -9,8 +9,12 @@ dictionary RouteHintHop {
  };
 
 [Error]
-enum SDKError {
-  "Error",
+enum SdkError {
```

Comment:
@ok300  Is the renaming from SDKError to SdkError affects the React Native implementation?

Reply:
Unliley, because I haven't found any references to `SDKError` (case-sensitive) anywhere.

Reply:
Searched generated RN files, no mention of `SDKError`.